src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -431,15 +431,32 @@ def consRecordAge(self, rse):
         isConsNewer = self.rseConsStats[rseName]['end_time'] > self.rseTimestamps[rseName]['prevStartTime']
         if not isConsNewer:
             msg = "RSE: %s With old consistency record in Rucio Consistency Monitor. " % rseName
-            msg += "Skipping it in the current run."
-            self.logger.info(msg)
-            raise MSUnmergedPlineExit(msg)
+            if rse['isClean']:
+                msg += "And the RSE has been cleaned during the last Rucio Consistency Monitor pooling cycle."
```

Comment:
`pooling` --> `polling`. Same for a few lines below.

Reply:
thanks ! :)

Reply:
It is always dangerous to use constructs like `if rse['isClean']` since you never know rse data type at run-time, it is much safer to use the following `if rse and 'isClean' in rse`. Here you always check that rse exists, and then check if certain key exists in a directionary.

Reply:
thanks @vkuznet ,  I was brave here because we make sure on several other places that we do have a common, well defined and universal `rse` data structure, which should always include that field. But you are correct, staying on the safe side and following good practice principles, this should be written the way you suggest here. I will fix that.